Tribler/Core/Utilities/tracker_utils.py

```diff
@@ -104,7 +104,7 @@ def parse_tracker_url(tracker_url):
         hostname, port = hostname_part.split(u':', 1)
         try:
             port = int(port)
-        except:
+        except ValueError:
             raise RuntimeError(u'Invalid port number.')
```

Comment:
I'm not sure whether we should throw a `RuntimeError` here. I think a `ValueError` would be closer the the actual error.